.github/workflows/consuming.yml

```diff
@@ -18,17 +18,32 @@ jobs:
       matrix:
         project: ['admiral', 'submariner', 'submariner-operator', 'lighthouse']
         deploytool: ['operator', 'helm']
+        cabledriver: ['libreswan']
```

Comment:
I think this isn't needed, can you try without it?

Reply:
I did try without, it is required the matrix var exists before `include`ing it.

Reply:
OK good to know since in their guide they seem to indicate you can create vars simply by including them